docs/beaver-builder/about-this-release.md

```diff
@@ -2,54 +2,147 @@
 
-In addition, prebuilt layout templates have been added to the free version. See [this article](/beaver-builder/layouts/templates/layout-templates-overview.md) for an overview of how to work with layout templates. Note that in the free version you can't save or export custom layout templates.
+<!--### Posts Slider & Posts Carousel modules
 
-## Accessibility improvements
+There's now an option to customize the output when no posts are found.
+-->
+### Pricing Table module 
 
-The following accessibility changes and improvements have been made:
+The Pricing Table module has a number of enhancements, summarized in this screenshot and explained in the following sections.
 
-* Added a **Screen reader text** field to all modules that allow icon selection, which assists screen readers in describing the icon correctly. See the article about [Icon and Icon Group modules](/beaver-builder/layouts/modules/icon-and-icon-group.md#icon-module) for an example of the HTML output.
-* Added labels to the [Subscribe Form module](/beaver-builder/layouts/modules/subscribe-form/subscribe-form.md#accessibility).
-* [Contact Form module](/beaver-builder/layouts/modules/contact-form/contact-form.md): Added an `aria-describedby` attribute to each input field to assist screenreaders in describing the purpose of the field. Allow both the spacebar and Enter keys to submit the form.
-* Removed blank lines from the [Callout Module](/beaver-builder/layouts/modules/callout-and-call-to-action.md).
-* Testimonials Module: Controls were flipped so the markup matches the display.
+![Pricing Table module 2.5 features](/img/2-5-features-pricing-table-overview.png) 
+
+#### Display a ribbon
+You can select an option to display a ribbon with custom text in the upper left or right corner of a price column.
+
+#### Monthly/yearly pricing
+There is a new **Toggle pricing** option where you can choose  a single price or a toggle to display monthly and yearly pricing.
+
+#### Icons for each feature
+You can assign icons to individual features. Icons added to an individual feature override a general icon if one has been set for all features.
+
+#### Typography options
+
+Each feature in a Pricing Box now has its own **Style** tab with the standard Beaver Builder [Typography section](/beaver-builder/styles/typography/typography.md). 
+
+Typography sections have also been added to the **Title** and **Price** in each pricing box and the **Toggle Pricing** icon at the top of the table.
+
+### Video module
+
+There are two additional options on the **Structured data** tab for `contentUrl` and `embedUrl`.
+
+## Column improvements
+
+* You can now set the width for single columns.  
+![Column width setting for single columns, 2.5 features](/img/2-5-features-single-column-width.png) 
+<!-- * There are now custom positions available for backgrounds. -->
```

Comment:
I commented this out because I didn't see any custom positions in the UI--if they're there would you let me know?